applications/luci-app-p910nd/root/usr/share/rpcd/acl.d/luci-app-p910nd.json

```diff
@@ -2,10 +2,17 @@
 	"luci-app-p910nd": {
 		"description": "Grant UCI access for luci-app-p910nd",
 		"read": {
-			"uci": [ "p910nd" ]
+			"uci": [ "p910nd" ],
+			"file": {
+				"/usr/bin/find -L /dev -maxdepth 3 -type c*": ["exec"],
```

Comment:
Please change the command pattern to exactly:

`/usr/bin/find -L /dev -maxdepth 3 -type c -name lp[0-9]`

This will prevent unwanted parameter injection.

Reply:
Changing it to exactly `/usr/bin/find -L /dev -maxdepth 3 -type c -name lp[0-9]` causes 403 when issuing the command via cgi-bin, and `Error executing "find" command: Access to command denied by ACL`. 



Changing it to exactly `/usr/bin/find -L /dev -maxdepth 3 -type c -name lp\[0-9\]` causes 403 and the page does not even load (prevented by 403).

I tried many permutations and could not find a specific one that worked.